### PR TITLE
Average time

### PR DESCRIPTION
**Pull request by geramirez**
adds average time scraper, csv of results, and updates yamls.

### CODE REVIEW

**Comment by khandelwal** [comment]
This build fails because 'scraper' is spelled incorrectly on this line: https://github.com/18F/foia/pull/56/files#diff-ec0b9a1a55a289ed9c738090fadbe5daR13

File: contacts/data/ABMC.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ departments:
   phone: 703-696-6897
   public_liaison: 'Michael G. Conley, Phone: (703) 696-6778'
   service_center: 'Phone: (703) 696-6897'
+  simple_average: '72'
```
Comment (khandelwal) [inline contacts/data/ABMC.yaml]: What units is this in? It might be useful to have it be part of the name perhaps. simple_average_days for example.

Comment (khandelwal) [inline contacts/data/ABMC.yaml]: Also, to be more precise, should it be: 

simple_request_processing_time_mean_days

File: contacts/html/FRTIB_2012_timedata.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
+
```
Comment (khandelwal) [inline contacts/html/FRTIB_2012_timedata.html:1]: I don't think we should be checking this in right?

File: contacts/data/ABMC.yaml (hunk 1 above)
Comment (geramirez) [inline contacts/data/ABMC.yaml]: Sure, I'll change the names.

File: contacts/html/FRTIB_2012_timedata.html (hunk 2 above)
Comment (geramirez) [inline contacts/html/FRTIB_2012_timedata.html:1]: @khandelwal I added it as testing data. I can take it out though.

**Comment by konklone** [comment]
I updated this branch to merge in the latest from `master` and resolved merge conflicts in 786f5b9.

@ramirezg, this looks just about ready for merging, but could you update the README to document how to use the new scraper and where it fits into the workflow?

(We're nearing or at the place where we should have a meta-script that runs all the scripts.)

**Comment by geramirez** [comment]
Verified that data in yaml files, csv, and foia.gov was consistent and checked NAs to make sure they were actually NAs.

**Comment by geramirez** [comment]
@jackiekazil Just updated with the changes we talked about yesterday.

**Comment by jackiekazil** [comment]
This is complete. I am merging into the main repo.

**Pull request merged**